<?xml version="1.0"?>
<launch>

	<!-- Launch arguments -->
	<arg name="grasp_params_1" default="$(find panda_softhand_control)/configs/grasp_params_1.yaml"/>
	
	<!-- Loading config file to param server -->
	<rosparam command="load" file="$(arg grasp_params_1)"/>

	<!-- Load main node -->
  	<node name="grasp_node_1" pkg="panda_softhand_control" type="grasp_robot_1" output="screen"/>	

	<!-- Temporarily publishing the object pose and tf. TODO: comment out when using vision -->
	<node name="rostopic_object_pose_node" pkg="rostopic" type="rostopic" args="pub -r 10 /grasp/chosen_object_1 geometry_msgs/Pose 
	'{position: {x: 0.7, y: 0, z: 0.055}, orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}}'"/>

	<node name="world_to_object_1" pkg="tf" type="static_transform_publisher" args="0.7 0 0.055 0.0 0.0 0.0 world object_1 100"/>


</launch>